cluster-service/Makefile

```diff
@@ -28,6 +28,7 @@ deploy:
 	ZONE_RESOURCE_ID=$(shell az network dns zone show -n ${ZONE_NAME} -g ${REGIONAL_RESOURCEGROUP} --query id -o tsv) && \
 	CX_SECRETS_KV_URL="https://${CX_SECRETS_KV_NAME}.vault.azure.net/" && \
 	CX_MI_KV_URL="https://${CX_MI_KV_NAME}.vault.azure.net/" && \
+	CX_SECRETS_KV_MI_CLIENT_ID=$(shell az aks show -n ${MGMT_AKS_NAME} -g ${MGMT_RESOURCEGROUP} --query 'addonProfiles.azureKeyvaultSecretsProvider.identity.clientId' -o tsv) && \
```

Comment:
Can you expand about this logic?

I am not sure that the identity to be used is the one directly associated to that 🤔 It depends on how the lifecycle team wants to design it: Is it the one directly associated to that? is it one dedicated for the secrets kv mi? is it shared? and so on